cmd/libsnap-confine-private/snap.c

```diff
@@ -355,3 +355,33 @@ void sc_snap_split_snap_component(const char *snap_component, char *snap_name, s
                                   char *component_name, size_t component_name_size) {
     sc_string_split(snap_component, '+', snap_name, snap_name_size, component_name, component_name_size);
 }
+
+char *sc_security_tag_to_unit_name(const char *security_tag) {
+    char unit_name[PATH_MAX] = {0};
+
+    const char *c = security_tag;
+    while (*c) {
```

Comment:
This would make a natural for loop, no? with


```suggestion
    for (const char *c = security_tag; *c != 0; c++) {
```

Reply:
applied